html/modules/custom/ncms_ui/src/Plugin/Field/FieldWidget/EntityReferenceBrowserTableWidgetNcms.php

```diff
@@ -120,6 +120,7 @@ public function buildTableRows(array $entities, $details_id, $field_parents) {
       $rowData[] = array_filter([
         'handle' => $this->isSortable() ? $this->buildSortableHandle() : NULL,
         'title-preview' => $this->getFirstColumn($entity),
+        'tags' => $this->getTagsColumn($entity),
```

Comment:
This should really only be about articles and I'd prefer to have a `getTags()` method on the `\Drupal\ncms_ui\Entity\Content\ContentBase` class, as that class already _knows_ about the computed tags field.